en/04_Changelogs/5.1.0.md

```diff
@@ -13,13 +13,16 @@ title: 5.1.0 (unreleased)
 ## Features and enhancements
 
 ### Other new features
+
 - You can now exclude specific `DataObject` models from the check and repair step of `dev/build` - see [ORM Performance](/developer_guides/performance/orm/#skip-check-and-repair) for more information.
 - You can change what `SearchFilter` the `TreeDropdownField` uses with yaml configuration - see [ORM Performance](/developer_guides/performance/orm/#treedropdownfield) for more information.
 - The [`i18nTextCollector`](api:SilverStripe\i18n\TextCollection\i18nTextCollector) now collects strings for ORM properties (e.g. `$db` fields) in `DataObject` and `Extension` classes, and from themes. See [i18n - collecting text](/developer_guides/i18n/#collecting-text) for more details.
 - Extensions which modify permissions for [`Group`](api:SilverStripe\Security\Group) records which return `true` will be respected, the same as when modifying permissions for any other `DataObject` record.
 
 ## Bug fixes
 
+- `SilverStripe\ORM\DataList::filterAny()` queries on many-many relations that use an aggregate `HAVING` clause now correctly use an `OR` conjunction rather than an incorrect `AND` conjunction.
```

Comment:
```suggestion
- [`DataList::filterAny()`](api:SilverStripe\ORM\DataList::filterAny()) queries on many-many relations that use an aggregate `HAVING` clause now correctly use an `OR` conjunction rather than an incorrect `AND` conjunction.
```

Reply:
Updated